website/components/consul-on-kubernetes-hero/style.module.css

```diff
@@ -0,0 +1,104 @@
+.ckHero {
+  background-position: top right;
+  background-repeat: no-repeat;
+  background-color: var(--black);
+  color: var(--white);
+  padding: 145px 0 163px;
+  overflow: hidden;
+
+  @media (--medium) {
+    padding: 78px 0 104px;
+  }
+
+  @media (--small) {
+    padding: 56px 0 80px;
+  }
+
+  & .headline {
+    flex-grow: 0;
+    flex-shrink: 0;
+    flex-basis: calc(50% - 16px);
+    max-width: 500px;
+    margin-left: auto;
+    margin-right: auto;
+    text-align: center;
+
+    @media (--large) {
+      text-align: left;
+      max-width: 384px;
+      margin: 0;
+    }
+
+    & .buttons {
+      display: flex;
+      flex-wrap: wrap;
+      align-items: center;
+      justify-content: center;
+
+      @media (--large) {
+        justify-content: flex-start;
+        margin: 0 -15px;
+      }
+
+      & .button {
+        margin: 0 15px;
+      }
```

Comment:
Probably a slightly more elegant way to handle this is with a `:not(:last-of-type)` query selector, and just apply margin-right to those who pass that.

Something like this:

```css
& .buttons {
  & .button:not(:last-of-type) {
    margin-right: 15px;
   }
}
```